clusterDeployer.py

```diff
@@ -103,6 +102,20 @@ def _all_hosts_with_workers(self) -> Set[ClusterHost]:
     is hardcoded to be the network hosting the API network.
     """
 
+    def remove_master_dhcp_entries(self, hosts: List[ClusterHost]) -> None:
+        for h in hosts:
+            h.prepare_remove_master_dhcp_entries(self._local_host.bridge)
+        # Restart libvirtd only once after removing all un-needed entries.
+        lh = host.LocalHost()
```

Comment:
We shouldn't create new connections.  Each cluster host has its own connection.  If we need to restart libvirtd we should be doing it inside the clusterHost method or even inside the VirtualBridge method.